fennel/gen/connector_pb2.pyi

```diff
@@ -579,6 +580,7 @@ class Source(google.protobuf.message.Message):
     cursor: builtins.str
     @property
     def lateness(self) -> google.protobuf.duration_pb2.Duration: ...
+    ts_sort_column: builtins.str
```

Comment:
Can we call this "timestamp_field" instead? That's more inline with terminology in other places. Also, the usage of this for sorting is only specific to some sources.